src/gps/gpsTask.c

```diff
@@ -64,6 +67,8 @@ void GPSTask(void *pvParameters)
                         if (result == GPS_MSG_SUCCESS) {
                                 const GpsSnapshot snap = getGpsSnapshot();
                                 lapstats_processUpdate(&snap);
+                                auto_logger_gps_sample_cb(&snap.sample);
```

Comment:
err... I meant, when triggering on other sensors we'd move it to a central location, like logger task.